docs/RELEASE.md

```diff
@@ -86,80 +108,36 @@ utilizes [`cargo-semver-checks`](https://github.com/obi1kenobi/cargo-semver-chec
+2. Create a new rc tag from the stable branch using [RC Automation flow](/.github/workflows/release-11_rc-automation.yml)
 
-1. Check and execute process [Clobbering](#clobbering), if needed.
-2. Check if there were any changes since the last release and abort, if not.
-3. Check out the latest commit of `stable`.
-4. Update the `CHANGELOG.md` version, date and compile the content using the PrDoc files.
-5. Open a Pull Request against `stable` for visibility of the release happening.
-6. Internal QA from the release team can happen here.
-7. Do a dry-run release to ensure that it *should* work.
-8. Comment in the Pull Request that a *Stable* release will happen from the merged commit hash.
-9. Release all changed crates to crates.io.
-10. Create the release `stableYYYYMMDD` on GitHub. Note that the Fellowship has a streamlined process that combines the
-    two last steps. A similar approach should be taken here.
-
-## Nightly Release
-
-Cadence: every day at 00:00 UTC+1. Responsible: Release Team
-
-This process aims to release the `master` branch as a *Nightly* release. The process can start at 00:00 UTC+1 and should
-automatically do the following steps.
-
-1. Check out the latest commit of branch `master`.
-2. Compare this commit to the latest `nightly*` tag and abort if there are no changes detected.
-3. Set the version of all crates that changed to `major.0.0-nightlyYYMMDD` where `major` is the last released `major`
-   version of that crate plus one.
-4. Patch the dependencies of the changed crates to point to the newest version of the dependency.
-5. Tag this commit as `nightlyYYMMDD`.
-6. Do a dry-run release to ensure that it *should* work.
-7. Push this tag (the commit will not belong to any branch).
-8. Release all crates that had changed to crates.io.
-
-## Clobbering
-
-Cadence: every 6th release (~3 months). Responsible: Release Team
-
-This process aims to bring branch `stable` in sync with the latest audited commit of `master`. It is not done via a Pull
-Request but rather by just copying files. It should be automated.  
-The following script is provided to do the clobbering. Note that it keeps the complete history of all past clobbering
-processes.
-
-```bash
-# Ensure we have the latest remote data
-git fetch
-# Switch to the stable branch
-git checkout stable
-
-# Delete all tracked files in the working directory
-git ls-files -z | xargs -0 rm -f
-# Find and delete any empty directories
-find . -type d -empty -delete
-
-# Get the last audited commit
-AUDITED=$(git rev-parse --short=10 origin/audited)
-# Grab the files from the commit
-git checkout $AUDITED -- .
-
-# Stage, commit, and push the working directory which now matches 'audited' 1:1
-git add .
-git commit -m "Clobbering with audited ($AUDITED)"
-git push
-```
-
-## Bug and Security Fix
-
-Cadence: n.a. Responsible: Developer
+From the forked polkadot-sdk repository in the [paritytech-release org](https://github.com/paritytech-release/polkadot-sdk/actions):
```

Comment:
I would add a 10th step regarding syncing templates, which are user facing. Similar for the latest stable release latest patch.